docs/api-reference-1/wordpress-source.md

```diff
@@ -259,6 +259,9 @@ The [`wp-source` package](https://github.com/frontity/frontity/tree/dev/packages
 
 ### Actions
 
+Actions don't return data. Data is always accessed via the state.
+That's because Frontity is following the [Flux pattern](https://facebook.github.io/flux/) (like Redux).
```

Comment:
Maybe this information should be in the "Learning Frontity - Actions" section?

Apart from that, maybe it's worth mentioning that, even though actions don't return data, they return a promise that resolves when the action is finished.

You can do something like this:

```js
await actions.source.fetch("/some-post");
```

which is useful when you need to access the new state just after calling the action:

```js
await actions.source.fetch("/some-post"); // <- Wait until we fetch "/some-post".
const somePost = state.source.get("/some-post"); // <- The data will exist.
```

This is of course not needed on React components, because:

- They are not async (you cannot use `await`).
- They re-render when the `state` accessed changes.

```js
const SomePost = ({ actions, state }) => {
  useEffect(async () => {
    // You can use await here if you pass an async function, but it's not required.
    await actions.source.fetch("/some-post");
  }, []);

  // The data will not exist at first, `data.isReady` will be false.
  // But then, it will rerender when `actions.source.fetch` is finished.
  const somePost = state.source.get("/some-post");

  // This will work just fine.
  return somePost.isReady ? <Post link="/some-post" /> : <Loading />;
};
```

What do you think?